src/components/Navbar.tsx

```diff
@@ -8,28 +8,33 @@ import WeatherSign from './WeatherSign/WeatherSign';
 
 interface Pages {
 	label: string;
+	title: string;
 	id: number;
 	href: string;
 }
 
 const pages: Pages[] = [
 	{
 		label: 'Home',
+		title: 'Home',
```

Comment:
why do you need these? Especially if the content is the same :)

Reply:
i need to have in the <a> element width of the title, as they are different, to calculate the width of the element and have the before pseudo be the same size to before:content-[attr(title)] before:block before:h-0 before:overflow-hidden before:invisible before:font-bold. not sure if i can use label for it..